actions/shared-node-cache/action.yml

```diff
@@ -5,26 +5,40 @@ inputs:
     description: 'Node version to use'
     required: false
     default: '20.x'
+  cache-cypress:
```

Comment:
I've added this option because we probably don't want to pull down the Cypress cache for jobs that aren't going to invoke Cypress.